runtime/tests/sharedclassapi/sharedClasses.c

```diff
@@ -0,0 +1,641 @@
+/*******************************************************************************
+* Copyright (c) 2017, 2023 IBM Corp.
```

Comment:
Pls fix the Copyright format to match the others.
`Copyright IBM Corp. and others 2017`